addons/website/static/src/interactions/full_screen_height.js

```diff
@@ -0,0 +1,46 @@
+import { registry } from "@web/core/registry";
+import { Interaction } from "@website/core/interaction";
+import { isVisible } from "@web/core/utils/ui";
+
+export class FullScreenHeight extends Interaction {
+    static selector = ".o_full_screen_height";
+    dynamicContent = {
+        "_window": {
```

Comment:
I'll do that if they become imported Symbols - which would not be very nice, because I still hope to be able to use them as starting points for selectors (e.g. "_modal div.brol")